CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add cluster names to readme (DR-2928)
 - Update Platform and Linked Projects sections of About page (DR-2972)
 
+### Added
+
+- Added html id elements to logo, learn more link, swimlands images, and swimlane item counts. (DR-2885)
+
+### Updated
+
+- Update version of NYPL Reservoir to 3.0.0 (DR-2924)
+- Updated the hero component to use a fallback image if the original featured image returns an error (DR-2925)
```

Comment:
Once you merge the 0.1.7 release branch into this branch, move this new block above it and under the next prerelease section.